test/unit/mdc-list/foundation.test.js

```diff
@@ -47,7 +47,7 @@ test('defaultAdapter returns a complete adapter implementation', () => {
     'removeAttributeForElementIndex', 'addClassForElementIndex', 'removeClassForElementIndex',
     'focusItemAtIndex', 'setTabIndexForListItemChildren', 'followHref', 'hasRadioAtIndex',
     'hasCheckboxAtIndex', 'isCheckboxCheckedAtIndex', 'setCheckedCheckboxOrRadioAtIndex',
-    'isFocusInsideList',
+    'notifyAction', 'isFocusInsideList',
```

Comment:
You added notifyAction but you need to remove followHref - the unit test is failing because of this